Modules/_testcapi/util.h

```diff
@@ -23,3 +23,10 @@
         assert(!PyErr_Occurred());          \
         return PyLong_FromSsize_t(_ret);    \
     } while (0)
+
+/* Marker to check that pointer value was set. */
```

Comment:
Note: I have a personal preference for ``// comments`` :-) But it's fine to have mixed style :-)